src/Components/Nav/ProductDetail/ContentLeft/ContentLeft.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import "./ContentLeft.scss";
+import Slider from "./Slider/Slider";
+import Preview from "./Preview/Preview";
+import ReviewList from "./ReviewList/ReviewList";
+
+class ContentLeft extends Component {
+  state = {
+    slideIndex: 0,
+  };
+
+  handleSlide = e => {
+    const { className } = e.target;
+    const {
+      leftData: { imgUrls },
+    } = this.props;
+
+    const { slideIndex } = this.state;
+    const btnType = className.split(" ")[1];
+    this.setState({
+      slideIndex:
+        btnType === "prev"
+          ? slideIndex === 0
+            ? imgUrls.length - 1
+            : slideIndex - 1
+          : slideIndex === imgUrls.length - 1
+          ? 0
+          : slideIndex + 1,
+    });
```

Comment:
삼항연산자를 중첩하여 사용하면, 코드 작성은 간편하겠지만, 가독성이 많이 떨어집니다-! 꼭 필요한 경우가 아니라면, 지양해주세요~
아래 코드 한번 참고해보시면 좋을 것 같습니다 👐
```suggestion
let newIndex = this.state.slideIndex;

if (btnType === 'prev') newIndex = slideIndex === 0 ? imgUrls.length-1 : slideIndex-1;
if (btnType === 'next') newIndex = slideIndex === imgUrls.length-1 ? 0 : slideIndex+1;

this.setState({
  slideIndex: newIndex
})
```

Reply:
가독성이 오히려 떨어지는군요 ㅠㅠ 수정하겠습니다!